modules/broker-amqp/src/main/resources/swagger.yaml

```diff
@@ -87,6 +95,14 @@ paths:
         type: integer
         required: true
         description: Identifier of the channel
+      - in: query
+        name: ifUsed
```

Comment:
The default operation should be to delete if not used, so I we make the parameter as ifUnsued the default value has to be true, this could be a problem for the cli since you always need to set the parameter --ifUnused for the default funcationality to work

Reply:
Can we cross-check with what we have for queue delete? It should go inline with that. Plus we can set the default for ifUnused to true as well right?

Reply:
we have --force-used, and --force-non-empty that's another reason to pick the word this way.
We can set it to true by default, but when you set the parameter, it sets --whatever it sets the value to true, rather than toggling the default value. If you want the value to be changed you need to say --ifUnused false. See http://jcommander.org/
Thus I think we shouldn't change it, WDYT?

Reply:
In swagger, for `DELETE /qeueues/{name}` ifUnused and ifEmpty are set to true by default. In CLI we can follow what we have for queue delete as well. WDYT?

Reply:
But then again for a user who uses both REST and CLI the the parameters are opposite, it's a bit confusing don't you think?

Reply:
I agree. We should stick to one. And within swagger, we should be consistent as well.